spec/suites/map/MapSpec.js

```diff
@@ -791,6 +791,36 @@ describe("Map", function () {
 		});
 	});
 
+	describe('#padding options types', function () {
+		beforeEach(function () {
```

Comment:
Perhaps we may use `before`, to execute this block just once before all included tests.

Reply:
I think this needs to be a `beforeEach`, like the other tests, because of the scope of the map object. It doesn't have any measurable performance impact, so I think we should keep it like this to be consistent.